<launch>
  <master auto="start"/>
  <param name="/trex/start_dir" value="$(find trex_pr2)/test/master/m2/no_arms"/>

  <!-- Common Actions -->
  <param name="/trex/enable_switch_controllers" value="false"/>
  <param name="/trex/enable_safety_tuck_arms" value="true"/>
  <param name="/trex/enable_move_base" value="false"/>
  <param name="/trex/enable_move_base_local" value="false"/>
  <param name="/trex/enable_set_laser_tilt" value="false"/>
  <param name="/trex/enable_base_state_publisher" value="false"/>

  <!-- Door Domain Actions -->
  <param name="/trex/enable_doors_tuck_arms" value="false"/>
  <param name="/trex/enable_detect_plug_on_base" value="true"/>
  <param name="/trex/enable_detect_door" value="false"/>
  <param name="/trex/enable_detect_handle" value="false"/>
  <param name="/trex/enable_grasp_handle" value="false"/>
  <param name="/trex/enable_unlatch_handle" value="false"/>
  <param name="/trex/enable_open_door" value="false"/>
  <param name="/trex/enable_touch_door" value="false"/>
  <param name="/trex/enable_push_door" value="false"/>
  <param name="/trex/enable_release_handle" value="false"/>
  <param name="/trex/enable_move_base_door" value="false"/>
  <param name="/trex/enable_check_path" value="false"/>

  <!-- Plug Domain Actions -->
  <param name="/trex/enable_plugs_untuck_arms" value="true"/>
  <param name="/trex/enable_detect_plug_on_base" value="false"/>
  <param name="/trex/enable_move_and_grasp_plug" value="true"/>
  <param name="/trex/enable_localize_plug_in_gripper" value="true"/>
  <param name="/trex/enable_stow_plug" value="true"/>
  <param name="/trex/enable_unplug" value="true"/>
  <param name="/trex/enable_plug_in" value="true"/>
  <param name="/trex/enable_move_base_local" value="false"/>
  <param name="/trex/enable_detect_outlet_coarse" value="false"/>
  <param name="/trex/enable_detect_outlet_fine" value="false"/>
  <param name="/trex/enable_doors_tuck_arms" value="true"/>
  <param name="/trex/enable_doors_touch_door" value="true"/>
  <param name="/trex/enable_doors_grasp_handle" value="true"/>
  <node pkg="trex_pr2" type="stub_ros_container" respawn="true" />
  <node pkg="trex_pr2" type="actions" respawn="true" />

  <param name="/trex/update_rate" value="10"/>
  <param name="/trex/input_file" value="master.cfg"/>
  <param name="/trex/path" value="$(find trex_pr2)/cfg:$(find executive_trex_pr2)/cfg"/>
  <param name="/trex/time_limit" value="20000"/>
  <param name="/trex/log_dir" value="$(find trex_pr2)/logs"/>
  <param name="/trex/play_back" value="0"/>
</launch>
